sandbox/sandbox/src/host_executor.rs

```diff
@@ -138,12 +184,12 @@ pub struct EnvironmentDefinitionBuilder<T> {
 impl<T> EnvironmentDefinitionBuilder<T> {
     fn add_entry<N1, N2>(&mut self, module: N1, field: N2, extern_entity: env::ExternEntity)
     where
-        N1: Into<Vec<u8>>,
-        N2: Into<Vec<u8>>,
+        N1: Into<String>,
+        N2: Into<String>,
     {
         let entry = env::Entry {
-            module_name: module.into(),
-            field_name: field.into(),
+            module_name: module.into().into(),
```

Comment:
Why do we need two into?

Reply:
`T: Into<String>` => `String` => `Vec<u8>`